test/dense.jl

```diff
@@ -182,6 +182,18 @@ end
     @test !ad.safe_mode
 end
 
+@testset "AutoTaylorDiff" begin
+    ad = AutoTaylorDiff{2}()
```

Comment:
```suggestion
    ad = AutoTaylorDiff(; order=2)
```
sorry I had missed this one, otherwise it should be good to go

Reply:
Ok done!